addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -29,7 +29,9 @@
                                         <t t-call="HasSequenceHoles"/>
                                     </div>
                                 </div>
-                                <t t-if="journal_type == 'bank' || journal_type == 'cash' || journal_type == 'sale' || journal_type == 'purchase'" t-call="JournalBodyGraph"/>
+                                <div class="row mt-auto">
+                                    <t t-if="journal_type == 'bank' || journal_type == 'cash' || journal_type == 'sale' || journal_type == 'purchase'" t-call="JournalBodyGraph"/>
```

Comment:
I know that's not the purpose of this task but `==` could be change for `===` or you can do something like that:
```suggestion
                                    <t t-if="['bank', 'cash', 'sale', 'purchase'].includes(journal_type)" t-call="JournalBodyGraph"/>
```